ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-ref.yaml

```diff
@@ -8,6 +8,7 @@ ref:
       memory: 600Mi
     limits:
       memory: 2Gi
+  termination_grace_period_seconds: 300
```

Comment:
Why setting 5 minutes here? Did you find a case where the compression of the tests results is taking more than the default 30s?

Reply:
Also the title of the PR does not seem in line with the original fix, which was about trapping the wrong signal

Reply:
I set 5 minutes to discuss the fine-tuned value in this PR. We can start by setting 30 seconds and if we think that this can be increased.

Reply:
Unless explicitly required to fix a specific issue I think it's better to keep the default value